Yoast/wordpress-seo
# HowTo Schema output

## Summary

This PR can be summarized in the following changelog entry:

* Updated HowTo schema output to reflect Google's new guidelines.

⚠️ This needs a block transform: I've added an `id` attribute to each step, so Google can link to steps, but haven't built the block transform. ⚠️ 

## Test instructions

This PR can be tested by following these steps:

* For schema output test instructions see #12905.

* For migrations: Create a how to block on `trunk` and save the post.
* Switch to this branch and build the JS.
* Refresh the page and make sure the block is shown. Make sure there is one JS console error regarding the HTML output of the block. This is expected.

## Quality assurance


Fixes #12905
